crossdock/server/endtoend.py

```diff
@@ -101,10 +106,19 @@ def tracers(self, tracers):
 
     @property
     def local_agent_sender(self):
+        try:
+            host, port_str = os.getenv('AGENT_HOST_PORT').split(':')
+            port = int(port_str)
+        except:
+            host = 'jaeger-agent'
+            port = DEFAULT_REPORTING_PORT
+            pass
+
         return LocalAgentSender(
-            host=os.getenv('AGENT_HOST', 'jaeger-agent'),
```

Comment:
so this was never used?

Reply:
I believe it is the wrong `env` variable. It should be `JAEGER_AGENT_HOST`. I will double check this.

Reply:
BTW seems to be only for crossdock, not a general client use case.

Reply:
Definitely never used in either Dockerfile not docker-compose.yml.